releasetasks/templates/mobile/candidates_fennec.yml.tmpl

```diff
@@ -40,7 +40,7 @@
                 - /bin/bash
                 - -cx
                 - >
-                  cd /home/worker/checkouts/gecko && ln -s /home/worker/artifacts artifacts && ./mach --log-no-times taskgraph decision --pushlog-id='0' --pushdate='0' --project='{{ branch }}' --message='try: -b o -p foo -u none -t none' --owner='asasaki@mozilla.com' --level='3' --base-repository=$GECKO_BASE_REPOSITORY --head-repository=$GECKO_HEAD_REPOSITORY --head-ref=$GECKO_HEAD_REF --head-rev=$GECKO_HEAD_REV --triggered-by='nightly' --target-tasks-method='candidates_fennec'
+                  cd /home/worker/checkouts/gecko && ln -s /home/worker/artifacts artifacts && ./mach --log-no-times taskgraph decision --pushlog-id='0' --pushdate='0' --project='{{ branch }}' --message='try: -b o -p foo -u none -t none' --owner='asasaki@mozilla.com' --level='3' --base-repository=$GECKO_BASE_REPOSITORY --head-repository=$GECKO_HEAD_REPOSITORY --head-ref=$GECKO_HEAD_REF --head-rev=$GECKO_HEAD_REV {% if branch == "mozilla-beta" %} --triggered-by='nightly' {% endif %} --target-tasks-method='candidates_fennec'
```

Comment:
I couldn't merge so I tweaked production locally to move past that point. Once the Github write access changes, I'll follow-up with a small fix in this PR and merge.